src/main/kotlin/com/novatecgmbh/eventsourcing/axon/admin/AxonAdministration.kt

```diff
@@ -0,0 +1,39 @@
+package com.novatecgmbh.eventsourcing.axon.admin
+
+import java.util.*
+import org.axonframework.config.EventProcessingConfiguration
+import org.axonframework.eventhandling.EventProcessor
+import org.axonframework.eventhandling.EventTrackerStatus
+import org.axonframework.eventhandling.TrackingEventProcessor
+import org.springframework.stereotype.Component
+
+@Component
+class AxonAdministration(private val eventProcessingConfiguration: EventProcessingConfiguration) {
+
+  fun resetTrackingEventProcessor(processingGroup: String) =
+      eventProcessingConfiguration.eventProcessorByProcessingGroup(
+              processingGroup, TrackingEventProcessor::class.java)
+          .ifPresent {
+            it.shutDown()
```

Comment:
Check it.supportsReset() prior resetting, see: https://github.com/AxonIQ/code-samples/blob/master/reset-handler/src/main/java/io/axoniq/framework/FrameworkEventProcessorRestController.java